Sources/SyrupCore/Stencil Extensions/SelectionSetExtension.swift

```diff
@@ -386,31 +417,12 @@ class SelectionSetExtension: Extension {
 				
 				fieldRenders.append(render)
 			}
-			
-			if let fragmentSpread = selection.fragmentSpread {
-				fragmentSpreadRenders.append(fragmentSpread.name)
-			}
-		}
-		
-		var addFragmentSpreadRenders = ""
-		var renderPassedTypeCondition = ""
-		
-		if let typeCondition = typeCondition {
-			renderPassedTypeCondition = ".map(x => copyWithTypeCondition(x, \(typeCondition)))"
```

Comment:
All this is no longer needed as this info is now encoded as part of the fragment selection.